vmdb/app/assets/javascripts/application.js

```diff
@@ -1,6 +1,6 @@
-//= require angular
 //= require jquery
 //= require jquery_overrides
+//= require angular
```

Comment:
this is actually done in #3184 and will remove when merged but so far this depends on angular.element being jQuerylike